src/lib.rs

```diff
@@ -30,6 +170,7 @@ mod driver;
 pub mod errors;
 mod gdal_major_object;
 mod metadata;
+#[doc(hidden)]
```

Comment:
Why do we need to hide this?

Reply:
Oops. Left over from an aborted experiment with documenting reexports.